guides/contexts.md

```diff
@@ -228,7 +228,7 @@ defmodule Hello.Accounts.User do
     field :name, :string
     field :username, :string
 
-    timestamps()
+    timestamps
```

Comment:
We definitely need to keep parens here as Elixir would emit a warning otherwise. You can remove parens for calls without args.

Reply:
I believe I followed ecto's docs with that so it's probably another docs inconsistency

Reply:
The ecto docs are likely just out of date. This code would emit a warning from v1.4.

Reply:
Should `timestamps: 0` be removed from Ecto’s `.formatter.exs` https://github.com/elixir-ecto/ecto/blob/master/.formatter.exs#L9 then?

Reply:
If that’s the case, maybe the formatter should emit a warning when a `<function>: 0` is included in the list of `locals_without_parens` :)

Reply:
It should because it has no effect.